ide/coqide.ml

```diff
@@ -400,12 +400,16 @@ let reset_autosave_timer () =
 
 (** Export of functions used in [coqide_main] : *)
 
+[@@@ocaml.warning "-32"]
 let forbid_quit () =
   try FileAux.check_quit File.saveall; false
   with FileAux.DontQuit -> true
 
+[@@@ocaml.warning "-32"]
 let close_and_quit = FileAux.close_and_quit
+[@@@ocaml.warning "-32"]
 let crash_save = FileAux.crash_save
+[@@@ocaml.warning "-32"]
```

Comment:
What's going on here?

Reply:
Conditional compilation (used only on OSX).

Reply:
>Conditional compilation (used only on OSX).

But aren't they unconditionally in the mli?

Reply:
I thought no, otherwise they wouldn't trigger the warning. Now, I haven't tried to move the to the IFDEF block, maybe that's what should be done.

Reply:
No, you're right, I messed up here too. Will fix.